dashboards-observability/.cypress/integration/VisualizationCharts/12_boxPlot_chart.spec.js

```diff
@@ -0,0 +1,248 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/// <reference types="cypress" />
+import {
+  delay,
+  TEST_QUERIES,
+  querySearch,
+  landOnEventVisualizations,
+} from '../../utils/event_constants';
+
+const numberOfWindow = 4;
```

Comment:
Could we move these constants to its corresponding constant file?

Reply:
@mengweieric your suggestion is accepted, but this is pending because of Issue-https://github.com/opensearch-project/observability/issues/896 
I request you please go through this issue and reply on this so accordingly all the charts can have constant default value and later we can move them to a constant file